packages/web-components/src/global/main.css

```diff
@@ -5,3 +5,30 @@ va-process-list li :is(h2, h3, h4, h5) {
   clear: none;
   padding-top: 0.3rem;
 }
+
+va-breadcrumbs li > a {
+  background-color: transparent;
+  color: var(--color-link-default);
+  display: inline-block;
+  padding: 0.2rem;
+  transition-duration: 0.3s;
+  transition-property: color, background-color, border-color;
+  transition-timing-function: ease-in-out;
+}
+
+va-breadcrumbs li > a:hover {
+  background: rgba(0, 0, 0, 0.05);
+}
+
+va-breadcrumbs li > a:focus {
+  outline: 2px solid var(--color-gold-light);
+  outline-offset: 0.2rem;
+}
+
+va-breadcrumbs li > a[aria-current='page'] {
+  color: var(--color-gray-dark);
+  cursor: default;
+  font-weight: bold;
+  pointer-events: none;
+  text-decoration: none;
+}
```

Comment:
We might have to come up with a cleaner solution if we continue to have web components with nested slot content. Maybe an additional css file in the component directory for these nested rules and import it here?

Reply:
I do like the idea of keeping the CSS intended for a component with that component. Also, you're right, if we have more of these types of components, I could see this file getting huge and unorganized.

Reply:
shame file 2.0 :sweat_smile: